ui/src/components/easter/hero/HeroContainer.js

```diff
@@ -1,105 +1,94 @@
 import 'react-modal-video/scss/modal-video.scss';
-import React, { Fragment, useState } from 'react';
+import React,{Fragment, useState} from "react";
 import ModalVideo from 'react-modal-video';
-import {
-  Container,
-  Text,
-  Box,
-  Flex,
-  VStack,
-  Icon,
-  LinkOverlay,
-  _hover,
-  LinkBox,
-} from '@chakra-ui/react';
+import { 
+  Container, 
+  Text, 
+  Box, 
+  Flex, 
+  VStack, 
+  Icon, 
+  LinkOverlay, 
+  _hover, 
+  LinkBox} from "@chakra-ui/react";
 
+ 
+ 
 const HeroContainer = () => {
-  const [isOpen, setOpen] = useState(false);
+  const [isOpen, setOpen] = useState(false)
   return (
     <Box>
-      <Box bgColor="white" height="auto"></Box>
+      <Box bgColor='white' height = 'auto'>
+    </Box>
+    
+  <Box 
```

Comment:
needs indent. please make sure formatting is correct. this is quite difficult to read and follow